pkg/commands/models/worktree.go

```diff
@@ -0,0 +1,56 @@
+package models
+
+import (
+	"fmt"
+	"github.com/go-errors/errors"
+	"io/fs"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+// Worktree : A git worktree
+type Worktree struct {
+	Id     int
+	Path   string
+	Branch string
+}
+
+func (w *Worktree) RefName() string {
+	return w.Name()
+}
+
+func (w *Worktree) ID() string {
+	return w.RefName()
+}
+
+func (w *Worktree) Description() string {
+	return w.RefName()
+}
+
+func (w *Worktree) Name() string {
+	return filepath.Base(w.Path)
+}
+
+func (w *Worktree) Main() bool {
+	return w.Id == 0
+}
+
+func (w *Worktree) Current() bool {
```

Comment:
I don't think this method should live on the worktree model just because I'd prefer model methods to simply work with the data on the model. It's harder to write tests if we want to assert on a certain worktree being the current worktree because we'd need to mock out `os.Getwd` in the constructor.

Reply:
Where do you suggest a better location would be?

Reply:
I'd chuck it in the helper